src/ExportHelper.php

```diff
@@ -127,4 +127,23 @@ public static function addQuotesToJsonKeys(string $input): string
 
         return $output;
     }
+
+    /**
+     * @param array<mixed, mixed> $mapping
+     */
+    public static function removeTypesInMappingKeys(array &$mapping): void
+    {
+        $final = [];
+        foreach ($mapping as $k => &$v) {
+            if (is_string($k)) {
+                // only $oid and $date has to be preserved
+                $k = preg_replace('/(?!.*\.\$oid)(?!.*\.\$date)(\.\$.*)/', '', $k);
```

Comment:
Nema sa to odstranovat iba z konca stringu?
\+ radsej by som vymenoval vsetky mozne hodnoty ...

Budem mat napr. `my.key.$foo.bar` a tento regexp mi to rozbije